### PR TITLE
NFS-Injector README

### PR DESCRIPTION
**Pull request by muddpunk**
So yeah.. Not sure if i've done this right.. but.. here it is :P

### CODE REVIEW

**Comment by cecpk** [comment]
Are there a shell based Installation? Don’t want to clickyclicky 😝

**Comment by acocalypso** [comment]
@cecpk Deal with clickycklicky

File: README_nfs-injector-MagiskModule.md [hunk 1]
```diff
@@ -0,0 +1,20 @@
+####Magisk module: NFS-Injector
+
+For those that might suffer with performance issues with their atv devices, for example, constant crashing, NFS Injector helps manage your ram and provides an overall better experience with performance. It will probably need more testing, but so far there are reports that even a 1gb model was able to scan for quests. 
+
+Not that the previous statment means that you should go and buy a ton of 1 gb models, because it's not recommended at all, some people just get sent the wrong models or have bad ram and this can help out so as your money was not a complete waste. However the device in question wouldn't even load the game properly before but yielded some results with the module installed.
+
+Another example, a 2gb model would freeze up completely and would require the power cord to be pulled out nearly every 10 minutes, now said device will run for hours before freezing up, if at all.
```
Comment (fosJoddie) [inline README_nfs-injector-MagiskModule.md]: Change this to:
Another example, some 2gb models would freeze up completely and would require the power cord to be pulled out nearly every 10 minutes. Now said device might run for hours or even stop freezing up completely.

File: README_nfs-injector-MagiskModule.md [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+Below is the link to the xda page to get more information which includes instructions on how to change the modes you can use, between Balanced, Gaming and Ultra.
```
Comment (fosJoddie) [inline README_nfs-injector-MagiskModule.md]: XDA in capital letters <3

**Comment by muddpunk** [comment]
I heard SHELLder shiny's are more likely to appear 🤷 but even then you have to clickyclicky 😂

**Review by fosJoddie — CHANGES_REQUESTED** [review]

**Comment by krzee** [comment]
> Are there a shell based Installation? Don’t want to clickyclicky 😝

If you can find a way, I will happily implement it. Thing is, the method for this would normally be via cache partition with twrp, but those dont work with our recovery.

**Pull request merged**